Contentstack/Stack.h

```diff
@@ -229,6 +229,47 @@ BUILT_ASSUME_NONNULL_BEGIN
 */
 - (void)sync:(void (^)(SyncStack * BUILT_NULLABLE_P syncStack, NSError  * BUILT_NULLABLE_P error))completionBlock;
 
+/**
+ The Initial Sync request performs a complete sync of your app data. This is done for seq id.
+ It returns all the published entries and assets of the specified stack in response.
+ The response also contains a sync token, which you need to store,
+ since this token is used to get subsequent delta updates later.
+
+     //Obj-C
+         [stack syncSeqIdInit:^(SyncStack * _Nullable syncStack, NSError * _Nullable error) {
+
+         }];
+
+     //Swift
+         stack.syncSeqIdInit({ ( syncStack:SyncStack, error: NSError) in
+ 
+         })
+
+@param completionBlock called synchronization is done.
+*/
+- (void)syncSeqIdInit:(void (^)(SyncStack * BUILT_NULLABLE_P syncStack, NSError  * BUILT_NULLABLE_P error))completionBlock;
```

Comment:
The function should be named `initSeqSync` or `initialSequenceSync`.

Reasoning: Including "id" in the function name may cause confusion for developers as it doesn't serve any purpose in this context. This function doesn't require any input from the developer.

@vkalta